node/node.go

```diff
@@ -271,6 +272,8 @@ func New(cfg *Config, version string) (*Node, error) { //nolint:gocyclo,funlen
 		log.Warnw("Failed to parse Juno version, will not warn about new releases", "version", version)
 	}
 
+	rpcHandler.SetReady()
```

Comment:
I wonder if we should set this in rpcHandler.Run()??

Reply:
Yes I think that will be better

Reply:
Updated this to setting rpcHandler ready in rpcHandler.run and updated tests so that they check readiness after rpcHandler.run